lucene/misc/src/java/overview.html

```diff
@@ -35,7 +35,7 @@ <h2>NativeUnixDirectory</h2>
 have to compile on your platform.
 
 <p>
-{@link org.apache.lucene.misc.store.NativeUnixDirectory} is a Directory implementation that bypasses the
+{@link org.apache.lucene.misc.store.DirectIODirectory} is a Directory implementation that bypasses the
```

Comment:
More of the javadocs below need to go away, too.

Reply:
Done.